error/error_handler.go

```diff
@@ -94,22 +95,60 @@ func EchoHTTPErrorHanderFunc(e error, c echo.Context) (bool, error) {
 	switch he.Code {
 	case http.StatusNotFound:
 		if !strings.Contains(c.Request().Header.Get("Content-Type"), "application/json") {
-			err = c.Render(he.Code, "errors/html/404", echo.Map{"stacktrace": fmt.Sprintf("%+v", e)})
+			// Get from env.json file.
+			html404File := viper.GetString("http.errorMessage.e404.html.file")
+			if html404File != "" {
+				err = c.Render(he.Code, html404File, echo.Map{"stacktrace": fmt.Sprintf("%+v", e)})
+			} else {
+				err = c.Render(he.Code, "errors/html/404", echo.Map{"stacktrace": fmt.Sprintf("%+v", e)})
+			}
 		} else {
-			err = c.JSON(he.Code, errorResp{
-				ErrorCode: RESOURCE_NOT_FOUND,
-				ErrorMsg:  he.Message,
-			})
+			// Get from env.json file.
+			e404 := viper.GetStringMap("http.errorMessage.e404")
+			if val, ok := e404["json"]; ok {
+				err = c.JSON(he.Code, val)
+			} else {
+				err = c.JSON(he.Code, errorResp{ErrorCode: RESOURCE_NOT_FOUND, ErrorMsg: he.Message})
+			}
 		}
 
 	case http.StatusMethodNotAllowed:
 		if !strings.Contains(c.Request().Header.Get("Content-Type"), "application/json") {
 			err = c.Render(he.Code, "errors/html/405", echo.Map{"stacktrace": fmt.Sprintf("%+v", e)})
 		} else {
-			err = c.JSON(he.Code, errorResp{
-				ErrorCode: METHOD_NOT_ALLOWED,
-				ErrorMsg:  he.Message,
-			})
+			// Get from env.json file.
+			e405 := viper.GetStringMap("http.errorMessage.e405")
+			if val, ok := e405["json"]; ok {
+				err = c.JSON(he.Code, val)
+			} else {
+				err = c.JSON(he.Code, errorResp{ErrorCode: METHOD_NOT_ALLOWED, ErrorMsg: he.Message})
+			}
+		}
+
+	case http.StatusInternalServerError:
+		// Send error event to sentry if configured.
+		if viper.GetBool("sentry.on") {
+			if hub := sentryecho.GetHubFromContext(c); hub != nil {
+				hub.CaptureException(he)
```

Comment:
maybe use `hub.CaptureException(he.Internal)` is better. this way the error's stack will be displayed on sentry.